linera-views/src/reentrant_collection_view.rs

```diff
@@ -326,6 +326,168 @@ where
     }
 }
 
+impl<C, W> ReentrantByteCollectionView<C, W>
+where
+    C: Context + Send + Clone + 'static,
+    ViewError: From<C::Error>,
+    W: View<C> + Send + Sync + 'static,
+{
+    /// Load multiple entry for write at once.
+    /// The entries in vector_short_key have to be all distincts.
+    /// ```rust
+    /// # tokio_test::block_on(async {
+    /// # use linera_views::memory::{create_test_context, MemoryContext};
+    /// # use linera_views::reentrant_collection_view::ReentrantByteCollectionView;
+    /// # use linera_views::register_view::RegisterView;
+    /// # use crate::linera_views::views::View;
+    /// # let context = create_test_context();
+    ///   let mut coll : ReentrantByteCollectionView<_, RegisterView<_,String>> = ReentrantByteCollectionView::load(context).await.unwrap();
+    ///   let vector_short_key = vec![vec![0,1],vec![2,3]];
+    ///   let view = coll.try_load_multi_entry_mut(vector_short_key).await.unwrap();
+    ///   let value1 = view[0].get();
+    ///   let value2 = view[1].get();
+    ///   assert_eq!(*value1, String::default());
+    ///   assert_eq!(*value2, String::default());
+    /// # })
+    /// ```
+    pub async fn try_load_multi_entry_mut(
```

Comment:
It seems like we could (and should) replace `multi_entry` by `entries` everywhere.
(Another options is `multiple_entries` but I don't think it's needed.)

Reply:
Ok, move to `multi_entries`.